### PR TITLE
Power 4.6 Direct install, restricted install and RN

### PR DESCRIPTION
**Pull request by ktania46**
Pull Request for power 4.6 
@vikram-redhat

### CODE REVIEW

**Comment by vikram-redhat** [comment]
@kalexand-rh to peer review.

File: release_notes/ocp-4-6-release-notes.adoc [hunk 1]
```diff
@@ -627,6 +627,77 @@ In the table below, features are marked with the following statuses:
...
+With this release, IBM Power Systems are now compatible with {product-title} {product-version}. See xref:../installing/installing_ibm_power/installing-ibm-power.adoc#installing-ibm-power_installing-ibm-power[Installing a cluster on IBM Power] or xref:../installing/installing_ibm_power/installing-restricted-networks-ibm-power.adoc#installing-restricted-networks-ibm-power_installing-restricted-networks-ibm-power[Installing a cluster on IBM Power in a restricted network].
+
+[discrete]
+====== Restrictions
+
+Note the following restrictions for {product-title} on IBM Power:
+
+* {product-title} for IBM Power Systems does not include the following Technology Preview features:
++
+** Container-native virtualization (CNV)
```
Comment (kalexand-rh) [inline release_notes/ocp-4-6-release-notes.adoc:642]: CNV has GA'd with the name "OpenShift Virtualization"

@vikram-redhat, will you double-check this list?

File: release_notes/ocp-4-6-release-notes.adoc [hunk 2]
```diff
@@ -627,6 +627,77 @@ In the table below, features are marked with the following statuses:
...
+** OpenShift Serverless(knative, FaaS integrations)
```
Comment (kalexand-rh) [inline release_notes/ocp-4-6-release-notes.adoc:643]:
```suggestion
** OpenShift Serverless (knative, FaaS integrations)
```

File: release_notes/ocp-4-6-release-notes.adoc [hunk 3]
```diff
@@ -627,6 +627,77 @@ In the table below, features are marked with the following statuses:
...
+
+* The following OpenShift Container Platform features are unsupported:
```
Comment (kalexand-rh) [inline release_notes/ocp-4-6-release-notes.adoc:645]:
```suggestion
* The following {product-title} features are unsupported:
```

File: release_notes/ocp-4-6-release-notes.adoc [hunk 4]
```diff
@@ -627,6 +627,77 @@ In the table below, features are marked with the following statuses:
...
+** Red Hat OpenShift Service Mesh(istio, jaeger, kiali)
```
Comment (kalexand-rh) [inline release_notes/ocp-4-6-release-notes.adoc:646]:
```suggestion
** Red Hat OpenShift Service Mesh (istio, jaeger, kiali)
```

File: release_notes/ocp-4-6-release-notes.adoc [hunk 5]
```diff
@@ -627,6 +627,77 @@ In the table below, features are marked with the following statuses:
...
+** CodeReady Workspaces
+** CodeReady Containers (CRC)
+** OpenShift Pipelines based on Tekton
+** OpenShift Container Platform Metering
```
Comment (kalexand-rh) [inline release_notes/ocp-4-6-release-notes.adoc:650]:
```suggestion
** {product-title} Metering
```

File: release_notes/ocp-4-6-release-notes.adoc [hunk 6]
```diff
@@ -627,6 +627,77 @@ In the table below, features are marked with the following statuses:
...
+** Multus Plugins (SR-IOV, IPVAN, Bridge with VLAN, Static IPAM)SR-IOV CNI plug-in
```
Comment (kalexand-rh) [inline release_notes/ocp-4-6-release-notes.adoc:651]:
```suggestion
** Multus Plugins (SR-IOV, IPVAN, Bridge with VLAN, Static IPAM)
** SR-IOV CNI plug-in
```

File: release_notes/ocp-4-6-release-notes.adoc (hunk 5 above)
Comment (kalexand-rh) [reply release_notes/ocp-4-6-release-notes.adoc:650]: This looks like a duplicate of line 654.

File: release_notes/ocp-4-6-release-notes.adoc [hunk 7]
```diff
@@ -627,6 +627,77 @@ In the table below, features are marked with the following statuses:
...
+** Red Hat Single Sign-On
+** JBoss Web Server (tomcat)
+** OpenShift Metering (Presto, Hive)
+
+
+* Worker nodes must run {op-system-first}.
+* Persistent storage must be of the `Filesystem` mode using local volumes, Network File System (NFS), OpenStack Cinder, or Container Storage Interface (CSI).
+* Networking must use either DHCP or static addressing with Red Hat Openshift SDN.
+* AdoptOpenJDK with OpenJ9
+* Installer Provisioned Infrastructure (IPI)
```
Comment (kalexand-rh) [inline release_notes/ocp-4-6-release-notes.adoc:661]:
```suggestion
* Installer-provisioned infrastructure
```

File: release_notes/ocp-4-6-release-notes.adoc [hunk 8]
```diff
@@ -627,6 +627,77 @@ In the table below, features are marked with the following statuses:
...
+[discrete]
+====== Supported Features
+
+* Currently, 3 operators are supported:
+** Cluster-Logging-Operator
+** Cluster-NFD-Operator
+** Elastic Search-Operator
+** Local storage operator
+
+* User Provisioned Infrastructure (UPI) deployment scenario on Bare-metal
```
Comment (kalexand-rh) [inline release_notes/ocp-4-6-release-notes.adoc:678]:
```suggestion
* User-provisioned infrastructure deployment scenario on bare-metal 
```

**Comment by jboxman** [comment]
@ktania46 is there any intersection with this https://github.com/openshift/openshift-docs/issues/26122?

**Comment by ktania46** [comment]
@SNiemann15

File: installing/installing_ibm_power/installing-ibm-power.adoc [hunk 9]
```diff
@@ -0,0 +1,90 @@
...
+
+[IMPORTANT]
+====
+Additional considerations exist for non-bare metal platforms. Review the information in the
+link:https://access.redhat.com/articles/4207611[guidelines for deploying {product-title} on non-tested platforms]before you install an {product-title} cluster.
+====
+
+.Prerequisites
+
+* Before you begin the installation process, you must move or remove any existing installation files. This ensures that the required installation files are created and updated during the installation process.
```
[the remark this answers is not in the document]
Comment (SNiemann15) [reply installing/installing_ibm_power/installing-ibm-power.adoc:19]: @kalexand-rh Yes thats whats meant there.

File: installing/installing_ibm_power/installing-restricted-networks-ibm-power.adoc [hunk 10]
```diff
@@ -0,0 +1,100 @@
...
+
+[IMPORTANT]
+====
+Additional considerations exist for non-bare metal platforms. Review the information in the
+link:https://access.redhat.com/articles/4207611[guidelines for deploying {product-title} on non-tested platforms] before you install an {product-title} cluster.
+====
+
+.Prerequisites
+
+* xref:../../installing/install_config/installing-restricted-networks-preparations.adoc#installing-restricted-networks-preparations[Create a mirror registry for installation in a restricted network] and obtain the `imageContentSources` data for your version of {product-title}. Use the `docker.io/ibmcom/registry-ppc64le:2.6.2.5` image.
```
[the remark this answers is not in the document]
Comment (SNiemann15) [reply installing/installing_ibm_power/installing-restricted-networks-ibm-power.adoc:19]: @kalexand-rh Neither Tania or me manage /do updates in installing-restricted-networks-preparations and we don't have z/Power specific content there.... we only link to it. I would think the writer wo has created those for the other architectures would do update checks.

**Comment by arcprabh** [comment]
@mcfadden

**Comment by vikram-redhat** [comment]
All peer reviews are done. QE has given ack for both Power and Z. I am merging this.

File: modules/installation-user-infra-machines-static-network.adoc [hunk 11]
```diff
@@ -46,31 +31,31 @@ ip=10.10.10.2::10.10.10.254:255.255.255.0:core0.example.com:enp1s0:none
 nameserver=4.4.4.41
 ----
 
-a|Specify multiple network interfaces by specifying multiple `ip=` entries.
-
+|Specify multiple network interfaces by specifying multiple `ip=` entries.
```
Comment (bobfuru) [inline modules/installation-user-infra-machines-static-network.adoc:34]: Not sure why these were dropped. The `a` is needed in some of these cells to properly display formatting.

File: modules/registry-configuring-storage-baremetal.adoc [hunk 12]
```diff
@@ -41,8 +69,9 @@ $ oc get pod -n openshift-image-registry
 [NOTE]
 =====
 * If the storage type is `emptyDIR`, the replica number cannot be greater than `1`.
-* If the storage type is `NFS`, you must enable the `no_wdelay` and `root_squash` mount options. For example:
-+
+* If the storage type is `NFS`, and you want to scale up the registry Pod by setting `replica>1` you must enable the `no_wdelay` mount option. For example:
```
Comment (bobfuru) [inline modules/registry-configuring-storage-baremetal.adoc:72]: We removed NFS examples in image registry docs. See these lines as example: https://github.com/openshift/openshift-docs/pull/26512/files#diff-4af87df61050e4b8fe830bbb6be46aaab303d94d8e0b79e1ede86f612413395dL72-L103

File: modules/installation-user-infra-machines-static-network.adoc [hunk 13]
```diff
@@ -7,32 +7,17 @@
...
-
-[IMPORTANT]
-====
-When adding networking arguments, you must also add the `rd.neednet=1` kernel argument.
-====
-
-The following table describes how to use `ip=`, `nameserver=`, and `bond=` kernel arguments for live ISO installs.
+If you install {op-system-first} from an ISO image, you can add kernel arguments
+when you boot that image to configure the node's networking.
+The following table describes and illustrates how to use those kernel arguments.
```
Comment (bobfuru) [inline modules/installation-user-infra-machines-static-network.adoc:12]: As noted in https://github.com/openshift/openshift-docs/pull/26559, I don't see why this text was changed. The original wording introduces the three options and tables we provide for advanced networking config in this section.

**Comment by sjstout** [comment]
LGTM! Thanks, Vikram!

**Pull request merged**